tests/fast-integration/eventing-test/utils.js

```diff
@@ -21,6 +21,8 @@ const kymaVersion = process.env.KYMA_VERSION || '';
 const isSKR = process.env.KYMA_TYPE === 'SKR';
 const skrInstanceId = process.env.INSTANCE_ID || '';
 const testCompassFlow = process.env.TEST_COMPASS_FLOW || false;
+const testV1Alpha2CRD = process.env.ENABLE_NEW_CRD_VERSION === 'true';
```

Comment:
Is this flag used for Subscription CRD only? If yes please rename ENV variable to something like ENABLE_SUBSCRIPTION_V1_ALPHA2.

Reply:
I have addressed you comment.